lib/openstax/aws/stack.rb

```diff
@@ -220,6 +222,18 @@ def apply_change_set(params: {}, wait: false)
       change_set
     end
 
+    def revert_to_previous_change_set(wait: false)
+      logger.info("**** DRY RUN ****") if dry_run
+
+      if @previous_parameters
+        logger.info("Applying previous parameters...")
```

Comment:
"Reverting to previous change set..." is probably more appropriate given the comment below about templates changing too